bugzilla_dashboard/components.py

```diff
@@ -0,0 +1,70 @@
+import sys
+import requests
+import urllib
+import structlog
+import json
+from bugzilla_dashboard.config import BZ_HOST
+from bugzilla_dashboard.config import COMPONENTS_URL
+
+logger = structlog.get_logger(__name__)
+
+
+# Get data from the give URL and transform return the result
+def getData(url):
+    try:
+        response = requests.get(url)
+        response.raise_for_status()
+        if response.status_code == 200:
+            return response.json()
+
+    except requests.exceptions.RequestException as e:
+        print(e)
+        sys.exit(1)
```

Comment:
Raise an exception or use `abort` from flask instead of crashing the whole process